README.md

```diff
@@ -7,11 +7,60 @@
 
 ## Overview
 
-This lesson is a very high-level over view of the React component lifecycle. We
-want to walk students through the whole shebang.
+As you've learned previously, a React component has two sets of properties: **props** and **state**. Props are given to the component by it's parent. You can think of props as an external influence that the component has no control over - like the weather. The other set of properties the component has is called the state. A component's state is internal to the component and it can change when the props changing or the user interacting with the component. For example, if the weather went from sunny to raining, you would put on a raincoat and wellies. In React component terms, your state could go from "wearing a jacket" to "wearing a raincoat and wellies". You know to put on a raincoat when it's raining because you've been told by your parents. A React component would have to be told what to do by the programmer - that's you! :)
 
-It might be helpful to break the cycle down into mounting (happens once),
-rendering (happens a bazillion times), and unmounting (happens once) phases.
+React components go through what we call a **component lifecycle**. This is broadly divided into three parts: **creation**, **updating** and **deletion**. The React framework was actually created by the social networking site Facebook because they needed a more *reactive* (see what they did there?) way to display information to their users. This means that every single thing you see on Facebook is actually a React component and/or a part of one! So for example, if you open a new chat window on, a ChatWindow component is **created**. As you are interacting with it and sending messages to your friends - that's the **updating** part. And when you finally close the window, the React component gets **deleted**. It seems all pretty straightforward from the user's perspective, however as you'll soon find out, there's a lot of stuff going on behind the scenes.
```

Comment:
> The React framework was actually created by the social networking site Facebook because they needed a more *reactive* (see what they did there?) way to display information to their users.

I think we can be a bit less verbose here, e.g. _React was created to create reactive UI's_ or something? Maybe it's also worth decoupling this from the specific example of Facebook, but keep the ChatBox analogy? I really like how you use that example to quickly illustrate the lifecycle. 👍